dotnet/samples/KernelSyntaxExamples/Example03_InlineFunctionDefinition.cs

```diff
@@ -7,7 +7,7 @@
 using RepoUtils;
 
 // ReSharper disable once InconsistentNaming
-public static class Example05_InlineFunctionDefinition
+public static class Example03_InlineFunctionDefinition
```

Comment:
At a minimum the files should be renamed as well.

Reply:
I'm going to put a hold on this PR until we are very close to v1 and have removed any invalid examples that may be still lurking around